test/fastify-instance.test.js

```diff
@@ -65,3 +66,11 @@ test('fastify instance get invalid ajv options.plugins', t => {
     }
   }))
 })
+
+test('fastify instance should contain default errorHandler', t => {
+  t.plan(3)
+  const fastify = Fastify()
+  t.ok(fastify[kErrorHandler] instanceof Function)
+  t.same(fastify.errorHandler, fastify[kErrorHandler])
+  t.same(Object.getOwnPropertyDescriptor(fastify, 'errorHandler').set, undefined)
+})
```

Comment:
Can you please add a test that when setting it inside a plugin the plugin gets the one that is set, while the external one stays the default one? Thanks

Reply:
Good point. Will do.